src/lib/netlist/build/makefile

```diff
@@ -151,7 +151,7 @@ OBJDIRS = $(OBJ) \
 			$(OBJ)/tests \
 			$(OBJ)/tools \
 			$(OBJ)/prg \
-			$(OBJ)/generated \
+			$(OBJ)/generated
```

Comment:
The reason for these ones is so that if an item is added or removed at the end of the list, only a single line changes.  Without the final continuation backslash, two lines need to change if an item is added or removed at the end of the list.

I don’t particularly like them though, as it’s somewhat error-prone.  Removing the blank line after the list will cause bad things to happen.